### PR TITLE
Update spec for Manual Update Control

### PR DESCRIPTION
**Pull request by yuindustries**

### CODE REVIEW

File: manual_update_control.md [hunk 1]
```diff
@@ -1,57 +1,66 @@
...
+
+## Future Goals
+
+### Force Update Single Resource / All Resources
+Sometimes Resources can get into a weird state, and users might want to re-trigger updates to try and fix the issue. 
+
+We can also offer the option of whether the user wants to trigger a Full Update or a LiveUpdate.
+
+### Pause/Unpause 
+A button to temporarily make everything Manual, which won’t persist past a web session. Tilt still “remembers” which items are persistently Manual, as configured in Tiltfile.
```
Comment (landism) [inline manual_update_control.md]: FWIW, when writing the original spec, I was imagining this as the most compelling way to use the feature:
1. picking out just a single resource to update seems like a rare use case
2. a double-tap of the pause/unpause key would serve as "update all"
3. pause/unpause allows users to switch between manual/automatic without changing the Tiltfile

File: manual_update_control.md [hunk 2]
```diff
@@ -1,57 +1,66 @@
...
+
+When you Unpause, Tilt updates everything that’s “dirty”.
+
+Use case: keep Tilt from freaking out when you do a Git checkout.
+
+### Keyboard Control
+Engineering culture tends to demand full keyboard control for tasks that are repetitive, and otherwise require a lot of sniping and clicking with a mouse.
```
Comment (landism) [inline manual_update_control.md]: Is this potentially cheap enough that we could just do it without waiting for user feedback? I feel like I'd be pretty unlikely to want to use this if I had to click every time to apply my changes, but maybe I'm unusual.

File: manual_update_control.md (hunk 1 above)
Comment (yuindustries) [reply manual_update_control.md]: This is an exceedingly good point. To the extent it even makes me a little nervous that I might have started with the wrong premise...but, I guess we'll collect feedback on this round and your PR comment can live here as evidence for a future "I told you so!"

File: manual_update_control.md (hunk 2 above)
Comment (yuindustries) [reply manual_update_control.md]: I definitely think we should prioritize Keyboard on the next sprint related to this feature.

**Pull request merged**